blueprints/component-addon/index.js

```diff
@@ -11,13 +11,13 @@ module.exports = {
   fileMapTokens: function() {
     return {
       __path__: function(options) {
-        if (options.pod) {
+        if (options.pod && options.inRepoAddon) {
```

Comment:
I made a mistake here - it's breaking the non-addon component pod generation.  I'll start looking for a way to detect if the generate is being called from an addon project.